Gemfile.lock

```diff
@@ -405,69 +409,68 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
       multi_json
     jwt (2.7.1)
     kubeclient (4.11.0)
       http (>= 3.0, < 6.0)
       jsonpath (~> 1.0)
       recursive-open-struct (~> 1.1, >= 1.1.1)
       rest-client (~> 2.0)
-    large_object_store (1.3.4)
-    llhttp-ffi (0.4.0)
+    large_object_store (1.7.0)
+      zstd-ruby (~> 1.5.5)
+    llhttp-ffi (0.5.0)
       ffi-compiler (~> 1.0)
       rake (~> 13.0)
-    lograge (0.12.0)
+    lograge (0.14.0)
       actionpack (>= 4)
       activesupport (>= 4)
       railties (>= 4)
       request_store (~> 1.0)
     logstash-event (1.2.02)
-    loofah (2.21.4)
+    loofah (2.22.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.12.0)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     marcel (1.0.2)
     marco-polo (2.0.1)
       railties (>= 6)
-    maxitest (3.3.0)
-      minitest (>= 5.0.0, < 5.12.0)
-    metaclass (0.0.4)
-    method_source (0.8.2)
-    mime-types (3.4.1)
+    maxitest (3.7.0)
+      minitest (>= 5.0.0, < 5.15.0)
+    method_source (1.0.0)
+    mime-types (3.5.2)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2023.0218.1)
+    mime-types-data (3.2024.0206)
     mini_mime (1.1.5)
     mini_portile2 (2.8.5)
-    minitest (5.11.3)
-    minitest-rails (6.1.0)
+    minitest (5.14.4)
+    minitest-rails (6.1.1)
       minitest (~> 5.10)
       railties (~> 6.1.0)
     mixlib-shellout (3.2.7)
       chef-utils
-    mocha (1.2.1)
-      metaclass (~> 0.0.1)
+    mocha (1.16.1)
     momentjs-rails (2.29.4.1)
       railties (>= 3.1)
-    msgpack (1.4.2)
+    msgpack (1.7.2)
```

Comment:
### [msgpack](http://msgpack.org/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.4.2 → 1.7.2

[_~~change-log~~, [source-code](https://github.com/msgpack/msgpack-ruby), [gem-diff](https://my.diffend.io/gems/msgpack/1.4.2/1.7.2)_]





<details>
<summary>Commits</summary>

A change of **243** commits. See the full changes on [the compare page](https://github.com/msgpack/msgpack-ruby/compare/v1.4.2...v1.7.2).

These are the first 10 commits:
- (0de0907) [spec description: Grammar fix preserve->preserves](https://github.com/msgpack/msgpack-ruby/commit/0de090717bd038ed2cdc3d75deb09d7bee4a6f0b)
- (1670227) [Setup GitHub Actions CI](https://github.com/msgpack/msgpack-ruby/commit/167022776e92938c526e300e59f9c6221b93bfbc)
- (c1547d3) [allow Truffle to fail](https://github.com/msgpack/msgpack-ruby/commit/c1547d31779d47ae9378da5aa684ec786095f82e)
- (2815689) [stop running TravisCI](https://github.com/msgpack/msgpack-ruby/commit/28156891c5a8b823638cd9ab2d7caa056aad9b0b)
- (4c3c467) [fix typo](https://github.com/msgpack/msgpack-ruby/commit/4c3c467c7bd49fd5c3c2249688f6a8f0e990bb14)
- (b946afd) [rename](https://github.com/msgpack/msgpack-ruby/commit/b946afdb1e46deed198a7b10d9e8517aa80bc21a)
- (2ea8b17) [suppress running duplicated CI for pull-requests](https://github.com/msgpack/msgpack-ruby/commit/2ea8b171c1ad4778d0a6e8b44b80e2d5fee94220)
- (7ebcef4) [allow failures for Truffle](https://github.com/msgpack/msgpack-ruby/commit/7ebcef46ed6e7d43431c6015dfedd2341c23a5fd)
- (0ed8386) [Merge pull request #209 from msgpack/Shopify-github-actions](https://github.com/msgpack/msgpack-ruby/commit/0ed838627101eb84c6288e9a13d69f5b88c4fc62)
- (aa9eeec) [Clean outdated feature checks](https://github.com/msgpack/msgpack-ruby/commit/aa9eeec5a68a628618cb39653f9bd7f08bc1e298)


</details>